### ISSUE TITLE
cheater?

### ISSUE THREAD

**Issue by futianyu7** [dituu/Semi-Siamese-Training#3]
lmdb_util.py and model.py not your creation. Please add the original author, ok?

**Comment by tiandunx** [comment]
I agree with @futianyu7 . You didn't write the source code in lmdb_util.py and model.py on your own.  Please respect the fruits of others' labor and add the author.

**Comment by xiaoboCASIA** [comment]
The original authors may have some words to say.

**Comment by cleardusk** [comment]
As one early author of the codebase, I agree with the comments above.

**Comment by jjwangx** [comment]
You "borrowed" code from other witout the permission from others. What's this action? ！Theif or Liar!!!
